src/components/NodeToolbar/AccessibilitySection/StyledFrame.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as React from 'react';
+import styled from 'styled-components';
+import colors from '../../../lib/colors';
+
+
+type Props = {
+  children: React.Element<*>,
+};
+
+
+const StyledFrame = styled.summary`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+  padding: .75rem .75rem 2px .75rem;
+  border: 1px solid ${colors.borderColor};
+  border-radius: 4px;
+  overflow: hidden;
+
+  &:before {
+    display: block;
+    position: absolute;
+    content: " ";
+    top: -8px;
+    left: 1em;
+    width: 12px;
+    height: 8px;
+    background: url(../NoseDetailsContainer.svg) no-repeat; /* TODO: decide on outline vs. color & add nose */
```

Comment:
0: This does not work under cordova, atm. But it was already like this before

Reply:
Yep! I've created a ticket for this: https://trello.com/c/9gIa1XlT/903-add-little-arrow-to-place-a11y-box